ci_example_python/package.xml

```diff
@@ -11,6 +11,9 @@
   <build_depend>rospy</build_depend>
   <run_depend>rospy</run_depend>
 
+  <build_depend>mpi_cmake_modules</build_depend>
+  <run_depend>mpi_cmake_modules</run_depend>
```

Comment:
Probably only needed as build depend?

Reply:
Surprisingly not... I am not sure why...

Reply:
Not related to your modification but I just noticed: rospy as build_depend is for sure not needed.